molpipeline/mol2any/mol2concatinated_vector.py

```diff
@@ -32,6 +32,7 @@ class MolToConcatenatedVector(MolToAnyPipelineElement):
     def __init__(
         self,
         element_list: list[tuple[str, MolToAnyPipelineElement]],
+        feature_names_prefix: Optional[str] = None,
```

Comment:
If you have a default MorganFP and a FeatureMorganFP both would get the same prefix, making them indistinguishable. 
I think it would make more sense to make this a dict. which maps the element name to a feature_prefix. But being honest, I would drop the feature_names_prefix.

Reply:
As discussed, we don't allow a custom user prefix. The parameter is now a bool flag, if True, the element names are used as prefix. If False, no prefix is used. During the generation of the names they are checked for uniqusness. We log a warning if there are duplicates.